pyccel/codegen/printing/pycode.py

```diff
@@ -284,7 +284,7 @@ def _print_IndexedElement(self, expr):
                 indices = indices[0]
 
             indices = [self._print(i) for i in indices]
-            if expr.pyccel_staging != 'syntactic' and isinstance(expr.base.class_type, HomogeneousTupleType):
+            if expr.pyccel_staging != 'syntactic' and isinstance(expr.base.class_type, (HomogeneousTupleType, HomogeneousListType)):
```

Comment:
Please update the CHANGELOG to mention this change

Reply:
can you check if the fix description needs more improvements ? 
0df73736f8062f68a843e3565f13bd4558469b9b